drivers/driver_linux.go

```diff
@@ -53,11 +53,11 @@ const (
 var (
 	// Slice of drivers that should be used in an order
 	priority = []string{
+		"overlay",
```

Comment:
This means we default to overlay in containers/storage, and overlay2-behavior is now default. 

Do you foresee any possibility to misunderstand this, and people ask "where did overlay2 go in containers/storage?"

Should we have a similar proposal for docker in that regard?